cli/cmd/check.go

```diff
@@ -90,6 +90,7 @@ func configureAndRunChecks(w io.Writer, options *checkOptions) error {
 		} else {
 			checks = append(checks, healthcheck.LinkerdPreInstallClusterChecks)
 		}
+		checks = append(checks, healthcheck.LinkerdPreInstallChecks)
```

Comment:
nit: there's no functional difference, but i find it helpful to order these the same way they are defined as constants (and output via `linkerd check`). mind moving `LinkerdPreInstallChecks` below `LinkerdPreInstallCapabilityChecks`?

Reply:
Ah, yep, fixed.